### PR TITLE
Composer 2.x compatibility

### PR DESCRIPTION
**Pull request by MekDrop**
[Composer 2.0 was released](https://blog.packagist.com/composer-2-0-is-now-available/) and this pull request lets to use your awesome plugin with new Composer version.

### CODE REVIEW

**Comment by mariusbuescher** [comment]
Nice to see, that you want to use this in your project. I did forget about this project already. :grin: It is pretty old though.

I will try to test it soon.

**Comment by mariusbuescher** [comment]
Unfortunately I can not test it atm, because this plugin currently does not support ARM devices.

File: src/NodeComposerPlugin.php [hunk 1]
```diff
@@ -126,4 +126,18 @@ public function onPostUpdate(Event $event)
...
+     * @inheritDoc
+     */
+    public function deactivate(Composer $composer, IOInterface $io)
+    {
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function uninstall(Composer $composer, IOInterface $io)
```
Comment (mariusbuescher) [inline src/NodeComposerPlugin.php:140]: A cleanup would be nice. Like remove all installed node versions.

Comment (MekDrop) [reply src/NodeComposerPlugin.php:140]: I aggree, but at the moment I would like not implement this by myself. There are many things todo. 

Could be this as some new planned feature?

**Comment by MekDrop** [comment]
I added pull request for CI tests (#5) , so with that you could test if everything ok.

**Pull request merged**